scripts/_cli.py

```diff
@@ -0,0 +1,207 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText:  PyPSA-Earth and PyPSA-Eur Authors
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+import textwrap
+
+import yaml
+from retrieve_databundle_light import *
+from rich.console import Console
+from rich.markdown import Markdown
+from rich.table import Table
+
+configfile = ["config.default.yaml", "configs/bundle_config.yaml", "config.yaml"]
+
+config = {}
+for c in configfile:
+    if os.path.isfile(c):
+        with open(c) as file:
+            config_append = yaml.safe_load(file)
+
+        config.update(config_append)
+
+
+def console_markdown(markdown, lvl=1):
+
+    console = Console()
+    for i in range(lvl):
+        markdown = textwrap.dedent(markdown)
+    md = Markdown(markdown)
+
+    return console.print(md)
+
+
+def console_table(dataframe, table_kw={}):
```

Comment:
I guess the whole this part (up to the line 146) is planned to be moved into `_cli` file, right? [I recognise that it's a draft, so apologies for a potentially pre-mature comment!]

Reply:
Thanks for revising the folders structure, and I think you are absolutely correct about using `__name__ == "__main__"` check as `databundle_cli` is intended for to be run as a stand-alone script.